integration_tests/cli/auth-demo.sh

```diff
@@ -129,6 +129,12 @@ check_response "$result" "\"code\": 0"
 
 test_divider
 
+echo "Alice doesn't reject account for \"$user\", because Alice already approved account"
```

Comment:
Better to say "Alice cannot reject account"

Reply:
Done! ✅